examples/flutter_gallery/lib/demo/material/slider_demo.dart

```diff
@@ -11,12 +13,114 @@ class SliderDemo extends StatefulWidget {
   _SliderDemoState createState() => new _SliderDemoState();
 }
 
+Path _triangle(double size, Offset thumbCenter, {bool invert: false}) {
+  final Path thumbPath = new Path();
+  final double height = math.sqrt(3.0) / 2.0;
+  final double halfSide = size / 2.0;
+  final double centerHeight = size * height / 3.0;
+  final double sign = invert ? -1.0 : 1.0;
+  thumbPath.moveTo(thumbCenter.dx - halfSide, thumbCenter.dy + sign * centerHeight);
+  thumbPath.lineTo(thumbCenter.dx, thumbCenter.dy - 2.0 * sign * centerHeight);
+  thumbPath.lineTo(thumbCenter.dx + halfSide, thumbCenter.dy + sign * centerHeight);
+  thumbPath.close();
+  return thumbPath;
+}
+
+class _CustomThumbShape extends SliderComponentShape {
+  static const double _thumbSize = 4.0;
+  static const double _disabledThumbSize = 4.0;
```

Comment:
It seems a little unusual to have two sizes and the tween below, if they have the same value. Maybe use 6.0 and 4.0?

Reply:
Also: the values are actually radii, so maybe _thumbRadius etc?